video_xblock/tests/unit/test_backends.py

```diff
@@ -335,3 +355,95 @@ def test_download_video_url_delegates_to_proper_xblock_attribute(self, backend,
 
         # Assert
         self.assertEqual(download_video_url, delegate_mock)
+
+
+@ddt
+class VimeoApiClientTest(VideoXBlockTestBase):
+    """
+    Test Vimeo backend API client.
+    """
+
+    def setUp(self):
+        super(VimeoApiClientTest, self).setUp()
+        self.vimeo_api_client = vimeo.VimeoApiClient(token='test_token')
+        self.vimeo_player = vimeo.VimeoPlayer(self.xblock)
+
+    @patch('video_xblock.backends.vimeo.requests.get')
+    def test_api_client_get(self, requests_get_mock):
+        """
+        Test Vimeo's API client GET method.
+        """
+        # Arrange
+        test_body = {'test': 'body'}
+        requests_get_mock.side_effect = [
+            ResponseStub(status_code=200, body=test_body),
+            ResponseStub(status_code=400)
+        ]
+
+        # Act
+        response = self.vimeo_api_client.get(url='test_url')
+
+        # Assert
+        requests_get_mock.assert_called_with('test_url', headers={'Authorization': 'Bearer test_token'})
+        self.assertEqual(response, test_body)
+        self.assertRaises(vimeo.VimeoApiClientError, self.vimeo_api_client.get, url='test_url')
+
+    def test_api_client_post(self):
+        """
+        Test Vimeo's API client POST method.
+        """
+        self.assertRaises(vimeo.VimeoApiClientError, self.vimeo_api_client.post, "test_url", "test_payload")
+
+    @data([{
```

Comment:
`@data` decorator is used to write parametrized tests.

Here you supply only one value which makes `@data` pointless. It's better to move this declaration down to Arrange section inside test function body.